vespalib/src/vespa/vespalib/datastore/unique_store_comparator.h

```diff
@@ -39,51 +39,61 @@ class UniqueStoreFloatingPointComparatorHelper
     }
 };
 
-/*
+/**
  * Specialized helper class for comparing float elements in unique store with
  * special handling of NAN.
  */
 template <>
 class UniqueStoreComparatorHelper<float> : public UniqueStoreFloatingPointComparatorHelper<float> {
 };
 
-/*
+/**
  * Specialized helper class for comparing double elements in unique store with
  * special handling of NAN.
  */
 template <>
 class UniqueStoreComparatorHelper<double> : public UniqueStoreFloatingPointComparatorHelper<double> {
 };
   
-/*
- * Compare two entries based on entry refs.  Valid entry ref is mapped
- * to an entry in a data store.  Invalid entry ref is mapped to a
- * temporary entry referenced by comparator instance.
+/**
+ * Compare two entries based on entry refs.
+ *
+ * Valid entry ref is mapped to an entry in a data store.
+ * Invalid entry ref is mapped to a temporary entry referenced by comparator instance.
```

Comment:
The temporary entry is now a part of the comparator instance, so the comment is outdated.

Reply:
Fixed